components/brave_rewards/common/pref_names.cc

```diff
@@ -34,7 +34,7 @@ const char kUseRewardsStagingServer[] = "brave.rewards.use_staging_server";
 const char kStatePromotionLastFetchStamp[] =
     "brave.rewards.promotion_last_fetch_stamp";
 const char kStatePromotionCorruptedMigrated[] =
-    "brave.rewards.promotion_corrupted_migrated";
+    "brave.rewards.promotion_corrupted_migrated2";
```

Comment:
we can just update this to 2 as we do the same thing that we do with version 1 and more